event/handlers.go

```diff
@@ -19,7 +19,7 @@ func KeyPressHandler(session *createwindow.Session, event *xproto.KeyPressEvent)
 	for _, browser := range session.ChromeList {
 		go func(browser createwindow.ChromeWindow) {
 			if err := chrometool.DispatchKeyEventToBrowser(browser.Ctx, chrometool.CSSSelector(browser.InputFieldSelector), str, mods, isFocussed); err != nil {
-				ErrorHandler(err)
+				ErrorHandler(err, false)
```

Comment:
When an argument is very opaque (hard to understand, add an argument comment:

`ErrorHandler(err, /*isFatal=*/false)`

Even better might be to just have two externally visible functions:

ShowFatalError(err)
ShowWarning(err)